### PR TITLE
Students API- part 1

### PR DESCRIPTION
**Pull request by SonaliBedge**
Students API- part 1

### CODE REVIEW

File: controllers/students.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  );
+  res.status(StatusCodes.OK).json({ students, count: students.length });
+};
+const getStudent = async (req, res) => {
+  res.send("get student detail");
+};
+const createStudent = async (req, res) => {
+  req.body.createdBy = req.user.userId;
+  const student = await Student.create(req.body);
+  res.status(StatusCodes.CREATED).json({ student });
```
Comment (ethanarns) [inline controllers/students.js:17]: Good use of named status codes

File: models/Student.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+const mongoose = require("mongoose");
+
+const StudentSchema = new mongoose.Schema(
+  {
+    SchoolName: {
+      type: String,
+      require: [true, "Please provide school name"],
+      maxlength: 200,
+    },
+    Grade: {
```
Comment (ethanarns) [inline models/Student.js:10]: Smart converting this to String, since there may be non-numerical grades

**Review by ethanarns — APPROVED** [review]
Looks good

**Comment by SonaliBedge** [comment]
Thank you for reviewing the assignment and for your comment.

**Pull request merged**